projects/plugins/jetpack/extensions/store/wordpress-com/types.ts

```diff
@@ -59,20 +74,13 @@ export type AiFeatureProps = {
 	errorMessage: string;
 	errorCode: string;
 	upgradeType: UpgradeTypeProp;
-	currentTier: {
-		value: TierValueProp;
-	};
+	currentTier: TierFreeProps | Tier100Props | Tier200Props | Tier500Props | TierUnlimitedProps;
```

Comment:
Maybe we could create a `TierProp` type, and reuse it:

Reply:
Yeah, I think I did that in the reverse order (enum and then realized a TierProp would come in handy)